src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -99,3 +99,94 @@ def generateRequestName(request):
     request["RequestName"] += "_%s_%s" % (currentTime, seconds)
     # then validate the final request name
     identifier(request["RequestName"])
+    
+        
+def protectedLFNs(requestInfo):
+    
+    reqData = RequestInfo(requestInfo)
+    result = []
+    if reqData.andFilterCheck(ACTIVE_STATUS_FILTER):
+        outs = requestInfo.get('OutputDatasets', [])
+        base= requestInfo.get('UnmergedLFNBase','/store/unmerged')
+        for out in outs:
+            _, dsn, ps, tier = out.split('/')
```

Comment:
Hmm, that wouldn't give the right answer. 
If you don't like not used variable, How about out.split('/')[1:]?